new-lamassu-admin/src/components/date-range-picker/Calendar.js

```diff
@@ -0,0 +1,144 @@
+import React, { useState } from 'react'
+import moment from 'moment'
+import { toInteger } from 'lodash/fp'
+import { makeStyles } from '@material-ui/core/styles'
+import Tile from './Tile'
+import { ReactComponent as Arrow } from '../../styling/icons/arrow/month_change.svg'
+import { primaryColor, zircon, fontSecondary } from '../../styling/variables'
+import typographyStyles from '../typography/styles'
+const { label2 } = typographyStyles
+
+const styles = {
+  navbar: {
+    display: 'flex',
+    justifyContent: 'space-between',
+    alignItems: 'center',
+    padding: [[15, 15]],
```

Comment:
pretty cool that this works, I have to adapt my styles and stop using string interpolation for this